corehq/apps/domain/models.py

```diff
@@ -67,24 +64,6 @@
         lang_lookup[lang['two']] = lang['names'][0]
 
 
-class DomainMigrations(DocumentSchema):
-    has_migrated_permissions = BooleanProperty(default=False)
-
-    def apply(self, domain):
-        if not self.has_migrated_permissions:
-            logging.info("Applying permissions migration to domain %s" % domain.name)
-            from corehq.apps.users.models import UserRole, WebUser
-            UserRole.init_domain_with_presets(domain.name)
```

Comment:
Just saw your earlier comment. I'd definitely move this to the function that creates domains, rather than doing it on save or something.